### ISSUE TITLE
Is it possible to get all user tracks using API?

### ISSUE THREAD

**Issue by volyx** [DevSnowflake/soundcloud-scraper#13]
Thank you for the library! It's really useful!

I'm trying to scrape all tracks from one user and could not find a way to get all tracks from user.

When I'm using Client.getUser(username) API I get only last ten tracks 

I use this call - https://soundcloud-scraper.js.org/#/docs/main/rewrite/class/Client?scrollTo=getUser

Example is here
https://github.com/volyx/gh-action-soundcloud-data-scraping/blob/5eabf6d993e020023e45d3401e6257382a33771a/data/2021-05-19.json#L14-L89

**Comment by twlite** [comment]
I dont think this is possible without using official api. This library currently gets those tracks from here:
![image](https://user-images.githubusercontent.com/46562212/118935078-d966d880-b96a-11eb-8d34-0065cf6150d6.png)
